Directory.Packages.props

```diff
@@ -16,7 +16,7 @@
     <_MicrosoftCodeAnalysisAnalyzersPackageVersion>3.11.0</_MicrosoftCodeAnalysisAnalyzersPackageVersion>
     <_RoslynDiagnosticAnalyzersPackageVersion>3.11.0-beta1.24508.2</_RoslynDiagnosticAnalyzersPackageVersion>
     <_MicrosoftVisualStudioLanguageServicesPackageVersion>$(MicrosoftVisualStudioLanguageServicesPackageVersion)</_MicrosoftVisualStudioLanguageServicesPackageVersion>
-    <_XunitPackageVersion>2.6.3</_XunitPackageVersion>
+    <_XunitPackageVersion>2.6.6</_XunitPackageVersion>
```

Comment:
Since these are open-source packages, should any of their version numbers be updated in NOTICE.txt? Check with @phil-allen-msft about this.

Reply:
Thanks, Phil is aware and will regen NOTICE.txt